src/LasagnaController.cpp

```diff
@@ -47,7 +47,7 @@ bool LasagnaController::update(Input input){
           launch_h = input.h_rel;
         }
         is_first_call = false;
-      } else if(input.h_rel - launch_h > constants.launch_h_thresh) state.status = ASCENT;
+      } else if(pasta_abs(input.h_rel - launch_h) > constants.launch_h_thresh) state.status = ASCENT;
```

Comment:
Let's say valbal is at 10km and falling when it boots up (with an `equil_h_thresh` of 11km) when the flight computer comes back online. It will be in the ASCENT state, rather than the EQUIL state as might be appropriate.

Is there any reason to believe it couldn't do a second equilibration?

Reply:
the different is pretty minor, so it doesn't really matter. The main different is that while the controller is in ASCENT, it will ballast more agressively if it needs too (so that it wont hit the ground). If it reboots in flight and goes back to ASCENT, the worst case scenario is that it ballasts a little more than it needs to, but even then it shouldn't over ballast too much. Not a big deal considering that there was in flight reboot in the first place